Utils/Editor/FindUnusedAssetsWindow.cs

```diff
@@ -0,0 +1,241 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace DUCK.Utils.Editor
+{
+	public class FindUnusedAssetsWindow : EditorWindow
+	{
+		private class Asset
+		{
+			public string Path { get; private set; }
+			public string Name { get; private set; }
+			public Object Obj { get; private set; }
+
+			public Asset(string path, Object obj)
+			{
+				Path = path;
+				Obj = obj;
+				if (obj != null)
+				{
+					Name = obj.name;
+				}
+			}
+		}
+
+		private const float WIDTH = 420;
+		private static bool includeVendor;
+		private static bool includePlugins;
+		private static bool includeDuck;
+		private static bool hasEditorLagBeenFound;
+		private static List<Asset> unusedAssets;
+		private static Vector2 scrollPosition;
+		private static FindUnusedAssetsWindow window;
+
+		[MenuItem("DUCK/Find Unused Assets")]
+		private static void Init()
+		{
+			window = (FindUnusedAssetsWindow) GetWindow(typeof(FindUnusedAssetsWindow));
+			window.Show();
+			window.maxSize = new Vector2(WIDTH, 1080);
+			window.minSize = new Vector2(100, WIDTH);
+			window.titleContent = new GUIContent("Find Unused Assets");
+
+			hasEditorLagBeenFound = true;
+		}
+
+		private static void Search()
+		{
+			List<string> usedAssets = FindUsedAssets();
+
+			if (usedAssets != null)
+			{
+				unusedAssets = FindUnusedObjects(FindAllProjectAssets(), usedAssets).OrderBy(a => a.Obj != null).ToList();
+				hasEditorLagBeenFound = true;
+			}
+			else
+			{
+				hasEditorLagBeenFound = false;
+			}
+		}
+
+		private void OnGUI()
+		{
+			EditorGUILayout.HelpBox("Does not include Scenes or Meta data.", MessageType.Info);
+			GUILayout.BeginHorizontal();
+			includeDuck = GUILayout.Toggle(includeDuck, "Include DUCK");
+			includePlugins = GUILayout.Toggle(includePlugins, "Include Plugins");
+			includeVendor = GUILayout.Toggle(includeVendor, "Include Vendor");
+			if (GUILayout.Button("Search"))
+			{
+				Search();
+			}
+			GUILayout.EndHorizontal();
+
+			if (!hasEditorLagBeenFound)
+			{
+				EditorGUILayout.HelpBox("Editor log not found! - You need to build your project before doing this!",
+					MessageType.Warning);
+			}
+			else if (unusedAssets != null)
+			{
+				GUILayout.Label(string.Format("Found {0} unused assets!", unusedAssets.Count), EditorStyles.boldLabel);
+				var scrollViewHeight = EditorGUIUtility.singleLineHeight * unusedAssets.Count;
+				var scrollViewPositionY = EditorGUIUtility.singleLineHeight * 6;
+				var scrollViewPosition = new Rect(0, scrollViewPositionY, WIDTH, window.position.height - scrollViewPositionY);
+				var scrollView = new Rect(0, 0, WIDTH, scrollViewHeight);
+				scrollPosition = GUI.BeginScrollView(scrollViewPosition, scrollPosition, scrollView, false, true);
+				for (var i = 0; i < unusedAssets.Count; i++)
+				{
+					var asset = unusedAssets[i];
```

Comment:
I could create a method for drawing each asset info, it would hopefully make this method more readable